make_timeline.py

```diff
@@ -139,7 +139,7 @@ def get_comms():
     """
     Get the list of comm passes from the DSN summary file.
     """
-    dat = yaml.load(open(DSN_COMMS_FILE, 'r'))
+    dat = yaml.load(open(DSN_COMMS_FILE, 'r'), Loader=yaml.BaseLoader)
```

Comment:
Per slack discussion, try using `yaml.safe_load` instead. That should be the standard replacement for `yaml.load`.

Reply:
Seems to work.  I couldn't tell from the docs which was actually safer?  The language of "probably safe" was interesting on https://pyyaml.org/wiki/PyYAMLDocumentation .  But this looks like a simpler replacement for sure so I'll use it as the standard replacement as it comes up.